src/blocks/modal.css

```diff
@@ -138,6 +142,34 @@
   position: relative;
   color: #000;
 }
+.modal__confirm_close-modal {
+  width: 430px;
+  height: 181px;
+}
+.modal__confirm_form {
+  width: 430px;
+  background-color: #fff;
+  border-radius: 10px;
+  position: relative;
+}
+.modal__confirm_close {
+  background-image: url(../images/closebutton.svg);
+  border: none;
+  width: 40px;
+  height: 40px;
+  background-color: transparent;
+  background-repeat: no-repeat;
+  background-size: contain;
+  position: absolute;
+  bottom: calc(100% + 5px);
+  left: calc(70% + 3px);
+  cursor: pointer;
+}
+
+.modal__confirm_container {
+  width: 430px;
+  height: 180px;
+}
 @media screen and (max-width: 630px) {
```

Comment:
1) The cursor of a disabled button should not be `pointer`, use `default`

Reference: https://developer.mozilla.org/en-US/docs/Web/CSS/cursor

2) Disabled button should not have `hover` state. 

There are several ways to fix that

1) use `not` selector in `.modal__button:hover` styles

```css
.modal__button:hover:not(:disabled) { // selects not disabled hovered button
    opacity: 0.8;
    cursor: pointer;
}
```

2) override hovered button styles (this rule should be after `.modal__button:hover`)

```css
.modal__button_disabled:hover { // selects disabled hovered button
    opacity: 0.2;
    cursor: default;
}
```